src/extensions/renderer/canvas/drawing-edges.js

```diff
@@ -208,8 +208,18 @@ CRp.drawEdgePath = function( edge, context, pts, type ){
       case 'straight':
       case 'segments':
       case 'haystack':
-        for( let i = 2; i + 1 < pts.length; i += 2 ){
-          context.lineTo( pts[ i ], pts[ i + 1] );
+        const taxiRadius= edge.pstyle( 'taxi-radius' )
+        const taxiRadiusValue = taxiRadius ? taxiRadius.value : 0
+
+        if( taxiRadiusValue > 0) {
+          for( let i = 2; i + 1 < pts.length; i += 2 ) {
+            context.arcTo( pts[ i - 2 ], pts[ i - 1 ], pts[ i ], pts[ i + 1 ], 2 * taxiRadiusValue )
+          }
+          context.lineTo( pts[ pts.length - 2 ], pts[ pts.length - 1 ] );
+        } else {
+          for( let i = 2; i + 1 < pts.length; i += 2 ) {
+            context.lineTo( pts[ i ], pts[ i + 1] );
+          }
```

Comment:
Note:  Taxi edges have an 'edge type' of 'segments', since they are currently rendered in the same way as segments -- but with the intermediate points calculated for you rather than specified manually.